### DISCUSSION TITLE
有关问题的讨论可以在 Discussion模块来讨论

### DISCUSSION THREAD

**Discussion by shirenchuang** [didi/KnowStreaming#649] [General]
有关问题的讨论可以在 Discussion模块来讨论！

也可以加微信 进KnowStreaming技术交流群

交流 KnowStreaming和kafka相关问题

微信：shiyanzu001    备注：进群

**Comment by 1123183721** [comment]
用docker装了下感觉好多bug，不敢用夫斯基

**Reply by shirenchuang** [reply]
> 用docker装了下感觉好多bug，不敢用夫斯基

有问题可以在社群里面提出来哈，我们会跟进的哈

**Reply by xuyf1001** [reply]
大佬，有没有企业版km的试用地址

**Comment by shirenchuang** [comment]
> 大佬，有没有企业版km的试用地址

企业版的都开源了
